spec/models/location_spec.rb

```diff
@@ -740,10 +740,38 @@
   describe "#units" do
     let(:location) { FactoryBot.build(:location) }
 
-    it "does add an error when the number of units is invalid" do
-      location.units = nil
-      location.valid?(:units)
-      expect(location.errors.count).to eq(2)
+    context "when the number of units is invalid" do
+      it "adds an error when units is nil" do
+        location.units = nil
+        location.valid?(:units)
+        expect(location.errors.count).to eq(2)
+      end
+
+      it "adds an error when units is 0" do
+        location.units = 0
+        location.valid?(:units)
+        expect(location.errors.count).to eq(2)
```

Comment:
I think this should be 1, cause the presence error shouldn't trigger

Reply:
Yup, changed